.github/workflows/test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       if: ${{ github.event_name != 'pull_request' || github.event.pull_request.head.repo.id == github.event.pull_request.base.repo.id }}
       with:
         access_token: ${{ github.token }}
-        
+
```

Comment:
What's the change here? My phone doesn't show anything, just want to confirm not adding spaces or something.

Reply:
It's removing whitespace (the same amount as the indent of the line above it).

Reply:
As waldyrious mentions, it's just removing whitespace that was included before. 👍🏻

Reply:
@MasterOdin, is this ok to merge?